src/mono/wasm/build/WasmApp.Common.targets

```diff
@@ -495,6 +495,7 @@
 
     <Error Condition="'$(WasmProfilers)' != '' and '$(EnableDiagnostics)' != 'false'" Text="%24(WasmProfilers)=$(WasmProfilers) is not supported together with %24(EnableDiagnostics)=$(EnableDiagnostics)"/>
     <Error Condition="'$(WasmProfilers)' != '' and '$(WasmPerformanceInstrumentation)' != ''" Text="%24(WasmProfilers)=$(WasmProfilers) is not supported together with %24(WasmPerformanceInstrumentation)=$(WasmPerformanceInstrumentation)"/>
+    <Error Condition="'$(WasmProfilers)' != '' and '$(RunAOTCompilation)' == 'true'" Text="%24(WasmProfilers)=$(WasmProfilers) is not supported together with %24(RunAOTCompilation)=$(RunAOTCompilation)"/>
```

Comment:
Does it mean that no profiles are supported in AOT mode or that this prop is used only for Deb tools profiler?

Reply:
Good catch, I only wanted to disable dev tools profiler, not log profiler.